src/OpenLoco/Windows/TownWindow.cpp

```diff
@@ -107,7 +107,7 @@ namespace OpenLoco::Ui::Windows::Town
             self->widgets[widx::demolish_town].right = self->width - 2;
             self->widgets[widx::demolish_town].left = self->width - 25;
 
-            if (isEditorMode())
+            if (isEditorMode() || isSandboxMode())
```

Comment:
This combo is used so much we should be making a common function for it so we don't miss a case.

Reply:
Suggestions for how to name it? `isEditorOrSandboxMode` seems a bit dull.

Reply:
`isGod()`

Reply:
Prepare for separate permissions like `user.canEditTowns()`, `user.canTerraform()`, etc? or `isCreative()`?

Reply:
I think implementing some sort of permission system is a good idea, but out of scope for this PR.

To be honest, I think neither `isGod` (after the 'god game' genre, presumably) nor `isCreative` are particularly intuitive with respect to what this check expresses. If we can't come up with a good name, I'd rather retain the expression as-is, to be honest.